src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar__start {
+  display: flex;
+  flex-direction: row;
+  width: $square * 7 + 10px;
+  flex-wrap: wrap;
+  margin: 200px auto;
+}
+
+@each $day in mon,tue,wed,thu,fri,sat,sun {
```

Comment:
It should move week days to collection variable like `$day: mon 100px, tue 200` and shrink `@each` block.